src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final String NAME = "circle";
```

Comment:
Here and in all other classes NAME is unnecessary
You may use getClass().getSimpleName() method to obtain same info